src/pages/index.js

```diff
@@ -1,93 +1,225 @@
+import PopupWithConfirm from "../components/PopupWithConfirm.js";
 
-const userInfo = new UserInfo(
-  selectors.profileTitle,
-  selectors.profileParagraph
-);
+// API
 
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (data) => {
-      const cardElement = createCard(data);
-      cardSection.addItem(cardElement);
-    },
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "9aa267fe-3315-41ab-a072-2d6dbd5ca77b",
+    "Content-Type": "application/json",
   },
-  selectors.cardList
-);
-
-const userInfoPopup = new PopupWithForm(selectors.editFormPopup, (data) => {
-  userInfo.setUserInfo(data);
 });
 
-const newCardPopup = new PopupWithForm(selectors.addFormPopup, (data) => {
-  const card = createCard(data);
-  cardSection.addItem(card);
-});
+let cardSection;
+
+api
+  .getAppInfo()
+  .then(([initialCards, userData]) => {
+    userInfo.setUserInfo(userData.name, userData.about);
+    userInfo.setAvatar(userData.avatar);
+    cardSection = new Section(
+      {
+        items: initialCards.reverse(),
+        renderer: (data) => {
+          const card = renderCard(data);
+          cardSection.addItem(card);
+        },
+      },
+      selectors.cardList
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+// Validation
 
-// VALIDATION //
 const formValidators = {};
 
 const enableValidation = (config) => {
   const formList = Array.from(document.querySelectorAll(config.formSelector));
   formList.forEach((formElement) => {
     const validator = new FormValidator(config, formElement);
     const formName = formElement.getAttribute("name");
-
-    formValidators[formName] = validator;
-    validator.enableValidation();
+    if (formName) {
+      formValidators[formName] = validator;
+      validator.enableValidation();
+    }
   });
 };
 
-//Event Listeners
-profileEditButton.addEventListener("click", () => {
-  fillProfileForm();
-  formValidators[selectors.profileForm].resetFormValidation();
-  userInfoPopup.open();
-});
+// Create Instances
 
-addNewCardButton.addEventListener("click", () => {
-  newCardPopup.open();
+const cardPreviewPopup = new PopupWithImage(selectors.previewPopup);
+
+const userInfo = new UserInfo(
+  selectors.profileTitle,
+  selectors.profileParagraph,
+  selectors.profileImage
+);
+
+const deleteCardPopup = new PopupWithConfirm(selectors.deleteCardPopup);
+
+const addCardPopup = new PopupWithForm(selectors.addFormPopup, (cardData) => {
+  addCardPopup.setLoading(true);
+  api
+    .createNewCard(cardData)
+    .then((res) => {
+      const card = renderCard(res);
+      cardSection.addItem(card);
+      addCardPopup.close();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      addCardPopup.setLoading(false);
+    });
 });
 
+const ProfileEditPopup = new PopupWithForm(
```

Comment:
`ProfileEditPopup`


Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)